fw/http2.c

```diff
@@ -485,7 +485,8 @@ tfw_h2_stream_xmit_prepare_resp(TfwStream *stream)
 
 	stream->xmit.h_len = resp->mit.acc_len;
 	stream->xmit.b_len = TFW_HTTP_RESP_CUT_BODY_SZ(resp);
-	if (test_bit(TFW_HTTP_B_CHUNKED, resp->flags))
+	if (test_bit(TFW_HTTP_B_CHUNKED, resp->flags)
+	    && !test_bit(TFW_HTTP_B_VOID_BODY, resp->flags))
```

Comment:
I'm wondering why do we set `TFW_HTTP_B_CHUNKED` for a bodyless response?

Reply:
We can avoid it because we use `TFW_HTTP_B_CHUNKED` flag to skip such connection specific headers and do not send it to client. And also we already have such checks in http.c
```
if (!(test_bit(TFW_HTTP_B_CHUNKED, hmresp->flags)
		      || test_bit(TFW_HTTP_B_VOID_BODY, hmresp->flags))
```
in tfw_http_resp_process

Reply:
OK, could you please add a comment that we can have Chuned encoded response with zero body?